nais/alerterator.yml

```diff
@@ -18,12 +18,18 @@ spec:
       description: "App \{{ $labels.app }} er nede i namespace \{{ $labels.kubernetes_namespace }}"
       action: "`kubectl describe pod \{{ $labels.kubernetes_pod_name }} -n \{{ $labels.kubernetes_namespace }}` for events, og `kubectl logs \{{ $labels.kubernetes_pod_name }} -n \{{ $labels.kubernetes_namespace }}` for logger"
 
-    - alert: høy feilrate i logger
+    - alert: høy andel error i logger
       severity: danger
-      expr: (100 * sum by (log_app, log_namespace) (rate(logd_messages_total{log_app="{{app}}",log_level=~"Warning|Error"}[3m])) / sum by (log_app, log_namespace) (rate(logd_messages_total{log_app="{{app}}"}[3m]))) > 10
+      expr: (100 * sum by (log_app, log_namespace) (rate(logd_messages_total{log_app="{{app}}",log_level=~"Error"}[3m])) / sum by (log_app, log_namespace) (rate(logd_messages_total{log_app="{{app}}"}[3m]))) > 0
       for: 3m
       action: "Sjekk loggene til app \{{ $labels.log_app }} i namespace \{{ $labels.log_namespace }}, for å se hvorfor det er så mye feil"
 
+    - alert: høy andel warning i logger
+      severity: warning
+      expr: (100 * sum by (log_app, log_namespace) (rate(logd_messages_total{log_app="{{app}}",log_level=~"Warning"}[3m])) / sum by (log_app, log_namespace) (rate(logd_messages_total{log_app="{{app}}"}[3m]))) > 100
```

Comment:
100 blir for høyt for prosesseringsappene tror jeg. Reduser til 10.

Reply:
10 på api, mottak og prosessering, eller kun prosessering?

Reply:
10 på alle de tenker jeg 👍

Reply:
Slik